src/zenml/cli/base.py

```diff
@@ -71,39 +80,38 @@ class ZenMLTemplate(StrEnum):
 )
 @click.option(
     "--template",
-    is_flag=True,
-    default=False,
+    type=click.Choice(list(ZENML_PROJECT_TEMPLATES)),
     required=False,
-    help="Use the ZenML starter project template to initialize the repository "
-    "and prompt the user to enter parameter values for the template.",
+    help="Use the ZenML projects templates to initialize the repository "
+    "and prompt to enter parameter values for the template.",
 )
 @click.option(
-    "--starter",
+    "--template-with-defaults",
     is_flag=True,
     default=False,
     required=False,
-    help="Use the ZenML starter project template to initialize the repository.",
+    help="Whether to use default parameters of the ZenML project template or prompt?",
```

Comment:
```suggestion
    help="Whether to use default parameters of the ZenML project template",
```

Prompting is what will happen anyway, I think?